src/test/java/uk/gov/companieshouse/service/DissolutionGetterTest.java

```diff
@@ -60,4 +69,43 @@ public void get_doesNotFindDissolution_returnsOptionalEmpty() {
 
         assertTrue(result.isEmpty());
     }
+
+    @Test
+    public void mapToDirectorPendingApproval_mapDirectorToBoolean_returnFalse_whenEmailNotFound() {
```

Comment:
you should be testing `isDirectorPendingApproval` rather than `mapDirectorsPendingApprovalToBoolean`